packages/create-refine-app/jest.config.js

```diff
@@ -0,0 +1,14 @@
+const { pathsToModuleNameMapper } = require("ts-jest/utils");
+const { compilerOptions } = require("./tsconfig");
+
+module.exports = {
+    preset: "ts-jest",
+    rootDir: "./",
+    name: "cli",
+    displayName: "cli",
+    setupFilesAfterEnv: ["<rootDir>/test/jest.setup.ts"],
+    testEnvironment: "jsdom",
```

Comment:
```suggestion
    testEnvironment: "node",
```
Can we update according to node env?